examples/banananation/build.gradle

```diff
@@ -13,15 +13,24 @@ test {
   useJUnitPlatform()
 }
 
+jar {
+  from {
+    configurations.runtimeClasspath.filter{ it.exists() }.collect{ it.isDirectory() ? it : zipTree(it) }
+  } {
+    exclude 'META-INF/LICENSE.txt'
+    exclude 'META-INF/NOTICE.txt'
+    exclude 'META-INF/LICENCE'
+  }
+}
+
```

Comment:
This `jar` section seems to have been added to `banananation`, but not the other example mission models.

Reply:
True. I'll go through and add it to the others, too.

Reply:
One more question - is this `jar` an optimization, or is it semantically significant? Should we advise users to add this to their own build.gradle files?

Reply:
It is necessary. If a model has dependencies that are not provided by its environment -- which is *typical*, since we can't predict all needs of all models -- it must include those dependencies in the JAR it ships to our execution platform.

This is true even beyond SPICE -- it just happened that our models have been simple enough to not have many dependencies -- and lucky enough that those it did have, like commons-lang3, were incidentally shared by the execution platforms it ran on.